### DISCUSSION TITLE
Matterbridge LXC: How IPv6 is ensured?

### DISCUSSION THREAD

**Discussion by Apollon77** [community-scripts/ProxmoxVE#430] [General]
I saw that there is a "Matterbridge LXC" script and I wonder how IPv6 works there, because matter uses IPv6 as mandatory requierement (IPv4 is optional) and my last information from proxmox was that the proxmox nodes do not get an IPv6 address and so also LXC can not have that ... How that works? ;-)

**Comment by Mellowlynx** [comment]
Not sure where you got that info.
But if you enable IPv6 on a node or LXC, it should get a IPv6 adres.

**Reply by Apollon77** [reply]
I was reading some proxmox forum entries to check about this and found "staff statements" there about this, thats why I thought it is set.

But ok now searching again there seems to be ways ...

https://forum.proxmox.com/threads/setting-up-ipv6.130114/

But so or so I think it makes sense to add a check to the install script of matterbridge LXC to verify that IPV6 is enabled and error else or point the user to a guide how toactivate. Else Matter experience will be problematic :-)
